README.md

```diff
@@ -32,9 +33,51 @@ You should see `Git LFS initialized` or a similar message. **Important**: if you
 
 Now, you can download the repository:
 
-- `git clone --recursive https://github.com/JoshParnell/ltheory.git ltheory`
+- `git clone https://github.com/Limit-Theory-Redux/ltheory.git`
 
-## Compiling
+## Compiling (Option 1: Precompiled Bin)
+
+You'll need need to download a precompiled bin folder.
+1. Go to https://github.com/Limit-Theory-Redux/ltheory/actions/runs/3620791750
```

Comment:
This is a specific action run that will become out of date over time. I suggest guiding users to the general actions page then selecting the latest successful build:
```suggestion
1. Go to https://github.com/Limit-Theory-Redux/ltheory/actions, then click the latest successful workflow run.
```